faucet/src/tests.rs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) 2019 Alain Brenzikofer
+// This file is part of Encointer
+//
+// Encointer is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Encointer is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Encointer.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Unit tests for the encointer_balances module.
+
+use super::*;
+use crate::mock::{Balances, EncointerFaucet, EncointerReputationCommitments, System};
+use codec::Encode;
+use encointer_primitives::{
+	ceremonies::Reputation,
+	communities::{Degree, Location},
+	reputation_commitments::{DescriptorType, FromStr},
+};
+use frame_support::{assert_err, assert_ok};
+use mock::{master, new_test_ext, RuntimeOrigin, TestRuntime};
+use sp_runtime::DispatchError;
+use test_utils::{helpers::*, storage::*, *};
+
+#[test]
+fn purpose_id_is_stored_correclty() {
```

Comment:
typo

Reply:
but: this test just tests the mock, not the dut. why is it here?
this test should be covered by the unit tests of reputation_commitments, right?